chapter2.adoc

```diff
@@ -54,6 +54,10 @@ S-interrupt file or an APLIC domain to a Supervisor Domain. The interface also
 describes CSRs to allow M-mode software to retain control on notification of
 interrupts when Supervisor domains are enabled.
 
+* `Smsdedbg` (<<Smsdedbg>>) - This extension defines the external debug
+configuration interface that is managed by M-mode software to opt-in to
+external debug when authorized by a RoT.
```

Comment:
The M-mode cannot determine whether debug is authorized or not. It should however not need to because Smsdedbg only indicates "Debug Allowed".

Reply:
Updating this to state - 
* `Smsdedbg` (<<Smsdedbg>>) - This extension defines the external debug
configuration interface that is managed by M-mode software to declare that
external debug is allowed for a supervisor domain. Whether external debug is
authorized or not is expected to be done via a RoT and is outside the scope
of this specification.